pg_search/src/lib.rs

```diff
@@ -43,6 +44,17 @@ const DEFAULT_STARTUP_COST: f64 = 10.0;
 
 pgrx::pg_module_magic!();
 
+extension_sql!(
+    r#"
+--
+-- ParadeDB is an Elasticsearch alternative built on Postgres. We're modernizing the features of
+-- Elasticsearch's product suite, starting with real-time search and analytics.
+--
+"#,
+    name = "bootstrap",
+    bootstrap
+);
+
```

Comment:
unnecessary

Reply:
Where does this get printed? I like it

Reply:
nowhere.  It would have just ended up at the top of the extension .schema.sql file.  I did this because at one point I needed to put something at the top of the schema file but then changed things around and forgot to remove this.

if we want to put a little header or whatever in there, we can do that in a different PR.  it's not necessary here.